firefox-ios/Client/Frontend/Settings/WebsiteDataManagement/WebsiteDataSearchResultsViewController.swift

```diff
@@ -37,6 +37,7 @@ class WebsiteDataSearchResultsViewController: ThemedTableViewController {
 
         tableView.isEditing = true
         tableView.allowsMultipleSelectionDuringEditing = true
+        tableView.register(cellType: ThemedCenteredTableViewCell.self)
```

Comment:
We don't need to register the cell here as it's already registered in `ThemedTableViewController`.